modules/apps/client-extension/client-extension-type-api/src/main/java/com/liferay/client/extension/type/manager/CETManager.java

```diff
@@ -31,8 +31,16 @@ public CET addCET(
 	public CET getCET(long companyId, String externalReferenceCode);
 
 	public List<CET> getCETs(
-			long companyId, String keywords, String type, Pagination pagination,
-			Sort sort)
+			long companyId, boolean excludeInstanceScopedCETs, String keywords,
+			String type, Pagination pagination, Sort sort)
+		throws PortalException;
+
+	public List<CET> getCETs(long companyId, String type)
+		throws PortalException;
```

Comment:
This is a new method, which is nice for some usages. However, its equivalent getCETsCount does not exist as it still takes a list of keywords. This makes the API a bit inconsistent because interface breaks the parity between the `getCETs` and `getCETsCount` counterparts, where we had:

- `getCETs(a, b, c, pagination, sort)`
- `getCETsCount(a, b, c)`   (parameters before pagination/sort remain the same)

Which becomes:

- `getCETs(a, b, c, d, pagination, sort)`
- `getCETsCount(a, b, c, d)`  (so far so good)
- `getCETs(a, b)`
- `getCETsCount(a, b, c)`  (this one breaks parity)

This is a minor nit, nevertheless, is there any reason to keep interface this way I am missing?

Reply:
Got it, you have a point. I'm thinking about keeping this method, so I will add a counter version for it

Reply:
@dsanz  Actually, there is only [one usage](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/frontend/data/set/provider/CETFDSDataProvider.java#L66) for getCETsCount. The shorter method is handy so the client code doesn't have to pass in a bunch of null and default values

Reply:
Do you think keeping this method without its count version is fine for now?

Reply:
I update the PR, kept the way it is because, until now, there is no need to have a count version of it.